Mathlib/FieldTheory/RatFunc.lean

```diff
@@ -1761,6 +1772,28 @@ theorem coe_X : ((X : RatFunc F) : LaurentSeries F) = single 1 1 := by
 set_option linter.uppercaseLean3 false in
 #align ratfunc.coe_X RatFunc.coe_X
 
+theorem single_pow {R : Type _} [Ring R] (n : ℕ) :
```

Comment:
In two of these three theorems, `single_pow` and `single_zpow`, the left side is an expression `single a b`, and the right side has other functions applied. In `single_inv`, this convention is reversed.
Should there be a convention, possibly even for `simp`, where the expression of the form `single a b` is either consistently the LHS or consistently the RHS?

Reply:
Good point. I do not know about other parts of the library, but at least I have aligned them here.

Reply:
I agree with the direction that you have aligned them to. Basically, we should see `single` as a homomorphism and try to make its arguments smaller.

Reply:
Taken the first one.